luxonis_ml/tracker/tracker.py

```diff
@@ -500,6 +525,43 @@ def upload_artifact(
                 )  # Stores details for retrying later
                 self.log_stored_logs_to_mlflow()
 
+    @rank_zero_only
+    def log_matrix(self, matrix: np.ndarray, name: str, step: int) -> None:
+        """Logs confusion matrix to the logging service.
```

Comment:
Let's remove "confusion" matrix from docstring as this is for general matrix object